.github/workflows/ci.yml

```diff
@@ -7,15 +7,18 @@ name: CI
 jobs:
   test:
     runs-on: ubuntu-latest
+    container: alpine:3.15
```

Comment:
your editor likes messing with whitespace :P Luckily github allows ignoring whitespace.

Reply:
yeah, that's the autoformatting. I use an indent of 2 spaces.  This seems to match what we use recently in Sourcegraph.